AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py

```diff
@@ -73,7 +72,7 @@ def GraphClosed_OnLevelChange():
     Report.result(Tests.graph_registered, graph_registered)
 
     # Open a different level, which should close any open Landscape Canvas graphs
-    general.open_level_no_prompt('WhiteBox/EmptyLevel')
```

Comment:
It looks like this was the only test left that was using this level, so can we delete `WhiteBox/EmptyLevel` now?

Reply:
I'm not sure, this test simply opened a level that was available at the time, and wasn't made for this test. I'll leave the level handling up to the Whitebox team when they handle their prefab test conversions.